kevm-pyk/src/kevm_pyk/summarizer.py

```diff
@@ -289,6 +289,35 @@ def get_todo_list() -> list[str]:
     return todo_list
 
 
+def stack_added(opcode_id: str) -> int:
```

Comment:
```suggestion
def stack_added(opcode: str) -> int:
```

(Similarly for the rest.)

Reply:
I'm working on the better rule id and will fix it latter!

Reply:
Done!

Reply:
Hey @Stevengre! Should the `-SUMMARY-USEGAS` labeling be in place already? For what I see in the last commit it's still using the old one (e.g., `rule [ADD-SUMMARY-0]:`)
Is this meant for a follow up PR? Thank you!

Reply:
Sure, I'll provide the better names in the next PR. Just after merging this one.

Reply:
For this one, I make the require clause more feasible for some opcode and reduce the rules to make it easier to generate better names.

Reply:
There are still some problems for some opcodes, like `balance`. But I think that I can resolve it when you need.